spec.html

```diff
@@ -5724,9 +5724,8 @@ <h1>
         1. If _argument_ is not an Object, return *false*.
         1. If _argument_ is an Array exotic object, return *true*.
         1. If _argument_ is a Proxy exotic object, then
-          1. If _argument_.[[ProxyHandler]] is *null*, throw a *TypeError* exception.
-          1. Let _target_ be _argument_.[[ProxyTarget]].
-          1. Return ? IsArray(_target_).
+          1. Let _proxyTarget_ be ? GetNonRevokedProxyTarget(_argument_).
```

Comment:
everywhere else you call it `_target_`, except here.

Reply:
Only in Proxy-specific algorithms. IsArray and GetFunctionRealm are not specific to Proxy objects, so I prefixed the alias name in them.